www/main/settings/components/demo.html

```diff
@@ -0,0 +1,46 @@
+<div id="slide10" class="slide">
+  <h1 class="gurbani gurmukhi">
```

Comment:
not sure the fonts are applying properly in the preview window. The Gurbani doesn't have the same font weight and such as the main presenter window.